doc/developer_guide/index.md

```diff
@@ -65,7 +65,6 @@ The version number of the package depends on [`git tags`](https://git-scm.com/bo
 ```bash
 git remote add upstream https://github.com/holoviz/panel.git
 git fetch --tags upstream
-git push --tags
```

Comment:
Why is this there in the first place? With this primarily being aimed at external contributors I would not expect them to even have permissions to push tags.

Reply:
It is from a fork, so they push to their forks not the main repo

https://holoviz-dev.github.io/panel/developer_guide/index.html#cloning-the-project

Reply:
I removed this because I got this 403 forbidden.
```
(base) panel $ git remote -v
origin	https://github.com/cdeil/panel.git (fetch)
origin	https://github.com/cdeil/panel.git (push)
upstream	https://github.com/holoviz/panel.git (fetch)
upstream	https://github.com/holoviz/panel.git (push)
(base) panel $ git push --tags
remote: Permission to holoviz/panel.git denied to cdeil.
fatal: unable to access 'https://github.com/holoviz/panel.git/': The requested URL returned error: 403
```
Not sure, maybe I cloned holoviz first few years ago and then added my fork and renamed remotes.

@hoxbro - is there a reason you want to keep this command here in the guide?

Naively I would think no-one needs to push tags out at that point of the workflow of doing local setup and should be OK to remove here?